src/ngAnimate/animateCss.js

```diff
@@ -232,6 +233,8 @@ var DETECT_STAGGER_CSS_PROPERTIES = {
   animationDelay:          ANIMATION_DELAY_PROP
 };
 
+var TRANSITION_EASING_VALUE = /ease|ease-in|ease-out|ease-in-out|linear|step-start|step-end|steps\(.+\)|cubic-bezier\(.+\)/;
```

Comment:
the regex fails when I have something like `transition: 1s ease-in-out color' (It will take the first match, which is `ease`).